### ISSUE TITLE
Activate flashlight from Argus pt

### ISSUE THREAD

**Issue by bloop16** [QuantumEntangledAndy/neolink#292]
Hy,

Is it possible to integrate a function to de - and activate the flashlight off an Reolink Argus PT?

Thank´s for your commitment to help us!

**Comment by QuantumEntangledAndy** [comment]
Is there anyway for an official client to do this?

**Comment by bloop16** [comment]
Yes it is possible in the official Reolink App for Windows and iOS. Android i don’t know. 
Symbolist with a Flashlight in the Liveview.

**Comment by QuantumEntangledAndy** [comment]
Oh yes I actually recall having this conversation with someone before. It's already been added into the mqtt controls

Use this message over mqtt

- `/control/floodlight [on|off]` Turns floodlight (if equipped) on/off

**Comment by bloop16** [comment]
Is it also possible to trigger over an SSL command? Im not using MQTT permanently because it still sucks out the battery.

**Comment by QuantumEntangledAndy** [comment]
Not at the moment. It's not too hard to add a command line command for it. I just won't have time for awhile
